lib/mix/lib/mix/tasks/new.ex

```diff
@@ -445,8 +445,8 @@ defmodule Mix.Tasks.New do
     use ExUnit.Case
     doctest <%= @mod %>
 
-    test "the truth" do
-      assert 1 + 1 == 2
+    test "salutates the world" do
+      assert <%= @mod %>.hello == :world
```

Comment:
the error trace message is

```shell
Assertion with =~ failed
code:  file =~ ~r"assert HelloWorld.hello() == :world"
left:  "defmodule HelloWorldTest do\n  use ExUnit.Case\n  doctest HelloWorld\n\n  test \"salutates the world\" do\n    assert HelloWorld.hello() == :world\n  end\nend\n"
right: ~r/assert HelloWorld.hello() == :world/
```

where the item on the `left` clearly includes `assert HelloWorld.hello() == :world`.